### ISSUE TITLE
Missing ``pygments`` Styles on RTD Hosted Docs

### ISSUE THREAD

**Issue by shawnbrown** [rtfd/sphinx_rtd_theme#633]
### Problem

Since the first week of May 2018 (coinciding with the release sphinx_rtd_theme 0.3.1), some projects on http://readthedocs.org no longer display syntax highlighting. While the live site no longer highlights code, the HTML and PDF downloads seem to be fine.

The HTML files **do** include the proper Pygments class attributes...

    <div class="code python highlight-default">
        <div class="highlight">
            <pre>
                <span></span><span class="n">html_theme</span> <span class="o">=</span> <span class="s2">&quot;sphinx_rtd_theme&quot;</span>
                <span class="n">html_theme_path</span> <span class="o">=</span> <span class="p">[</span><span class="s2">&quot;_themes&quot;</span><span class="p">,</span> <span class="p">]</span>
            </pre>
        </div>
    </div>

...but builds on http://readthedocs.org no longer include any CSS styles for these classes.

These styles used to be included in ``theme.css`` but they were removed: https://github.com/rtfd/sphinx_rtd_theme/commit/c43d473cb576ac2d10007d4baf45a4cdf960c38d#diff-8b5d5b0b20fffbc206fe1081243fb125L4. Locally, my builds include ``pygments.css`` but this file is not being included on the live site.

### Reproducible Project

I first noticed this with my own docs but I see that this is also happening in the sphinx-rtd-theme docs too:

- http://datatest.readthedocs.io/en/stable/tutorial/querying-data.html#loading-data
- https://sphinx-rtd-theme.readthedocs.io/en/latest/installing.html#via-git-or-download

#### Error Logs/Results

My builds are passing so I don't think this has anything to do with a change in my own ``conf.py``:

- https://readthedocs.org/projects/datatest/builds/

### Environment Info

Since this occurs on http://readthedocs.org/, I don't have access to the environment details but it does appear to be using sphinx-rtd-theme 0.3.1.

**Comment by Blendify** [comment]
@agjohnson seems like an error on RTD.

**Comment by stsewd** [comment]
Maybe is the same issue as https://github.com/rtfd/readthedocs.org/issues/4050?

**Comment by jessetan** [comment]
Your `latest` docs do have syntax highlighting, but your` stable` does not. This looks like it was fixed by pinning to a newer version of the theme.

**Comment by shawnbrown** [comment]
@jessetan, yes I pinned the new version (as mentioned in the link @stsewd provided) and changed my ``pygments_style`` property.

Here are the specific changes I made:
- In the documentation's **conf.py** file, I changed ``pygments_style = 'sphinx'`` to ``pygments_style = 'default'``.
- In the project's **requirements.txt**, I changed ``sphinx_rtd_theme`` to ``sphinx_rtd_theme>=0.3.1``.

This seems to have worked for me. I'll leave closing this issue up to your discretion--I wouldn't have opened a new issue, myself, if I had seen the closed one in the readthedocs.org repo so maybe it's good to keep this open until the updated theme is used by default on Read the Docs.
